webpack.config.js

```diff
@@ -77,7 +77,8 @@ const config = {
   node: {
       net: 'empty',
       tls: 'empty',
-      dns: 'empty'
+      dns: 'empty',
+      fs: 'empty'
```

Comment:
Just for my understanding can you briefly explain why this change was needed? Thanks!

Reply:
this is came from `dev` branch
https://github.com/Paratii-Video/paratii-portal/blame/dev/webpack.config.js#L81

Reply:
If it was still on `dev` it wouldn't be showing up in this diff 😸 

Maybe you need to re-merge `dev` again?